### DISCUSSION TITLE
Boot image from url

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#822] [Q&A]
Hey, is it possible to load a boot iso from a url? And if so, what is the url parameter for it?

**Comment by spetterman66** [comment]
why can't you run your own custom setup? it's the "setup" box below the built in profiles

**Comment by randompersononinternet69** [comment]
He meant to say "How can I boot a disk image over a url using the setup box? if I can, is there a parameter for it?"
________________________________

**Comment by CorruptionVR** [comment]
ok, so when I use the parameter `?cdrom.url=`, it gets stuck on `Downloading images [2/3]` whenever i use an image not from https://k.copy.sh

**Reply by SuperMaxusa** [reply]
Most likely because CORS is enabled on the server hosting this image, v86 gets stuck on `Downloading images`
https://github.com/copy/v86/issues/436#issuecomment-812887137
